### PR TITLE
Make poll interval configurable

### PR DESCRIPTION
**Pull request by keshav-dataco**

### CODE REVIEW

**Comment by keshav-dataco** [comment]
https://github.com/uber/athenadriver/issues/55

File: go/connection.go [hunk 1]
```diff
@@ -432,7 +433,7 @@ WAITING_FOR_RESULT:
 			obs.Scope().Timer(DriverName + ".query.StopQueryExecution").Record(timeStopQueryExecution)
 			obs.Log(ErrorLevel, "query canceled", zap.String("queryID", queryID))
 			return nil, ctx.Err()
-		case <-time.After(PoolInterval * time.Second):
+		case <-time.After(pollInterval):
```
Comment (henrywoo) [inline go/connection.go:436]: Can you add some unittest case for the new code/functions? Otherwise the code coverage will drop. Thank you. @keshav-dataco

Comment (keshav-dataco) [reply go/connection.go:436]: @henrywoo Done

Please review and merge if you are happy

**Review by henrywoo — APPROVED** [review]

**Comment by keshav-dataco** [comment]
@henrywoo  Is there anything pending from my side to get a new release out?

**Pull request merged**